doc/classes/AudioStreamPlayer2D.xml

```diff
@@ -139,30 +144,40 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="area_mask" type="int" setter="set_area_mask" getter="get_area_mask">
+			Areas in wich this sound plays.
 		</member>
 		<member name="attenuation" type="float" setter="set_attenuation" getter="get_attenuation">
+			Dampens Audio over distance with this as an exponent.
 		</member>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
```

Comment:
Add default value

Reply:
should i add default values everywhere? :)

Reply:
Not needed everywhere, but there are a lot of properties where it's helpful to know what the default behavior is, so you know whether you need to change it or not.  For example, if autoplay were on by default, that would be useful to know.